pkg/sql/plpgsql/parser/plpgsqllexbase/.gitignore

```diff
@@ -0,0 +1,4 @@
+# generated by ../parser/Makefile
+keywords.go
```

Comment:
Now I'm actually curious how all these three files are being generated. I doubt your change in `sql/lexbase/allkeywords` would make that happen. Could you verify ? :)